packages/patternfly-4/react-table/src/components/Table/utils/headerUtils.tsx

```diff
@@ -93,12 +93,12 @@ const mapHeader = (column: ICell, extra: any, key: number, ...props: any) => {
  * @param {*} extraObject with onSelect callback.
  * @returns {*} object with empty title, tranforms - Array, cellTransforms - Array.
  */
-const selectableTransforms = ({ onSelect }: { onSelect: OnSelect}) => [
+const selectableTransforms = ({ onSelect, canSelectAll }: { onSelect: OnSelect, canSelectAll: boolean}) => [
   ...(onSelect
     ? [
         {
           title: '',
-          transforms: [selectable],
+          transforms: (canSelectAll && [selectable]) || null,
           cellTransforms: [selectable]
```

Comment:
this seems correct - if we have a consumer that passes `onSelect` callback with `canSelectAll` set `false`, they'd receive the selectable cell transform (in the rows), but they'd not receive the "select all" header transform. 

I feel like another example showing that use case in Table.md might be warranted if you have time, but this appears correct to me!

Reply:
Yeah I'll throw in a checkbox to toggle it on and off in the UI of the docs, it will make it easier to see without needing to edit the code section.